### PR TITLE
Imporving `basic_patterns.rst` documentation file

### PR DESCRIPTION
**Pull request by RezaBehzadpour**
A couple of improvements in the `basic_patterns.rst` documentation file.

### CODE REVIEW

File: docs/source/basic_patterns.rst [hunk 1]
```diff
@@ -32,7 +32,9 @@ So, in this case, we are doing some more stuff. After we spawn *Alice* and
 
 First, we make *Alice* bind:
 
-addr = alice.bind('PUSH', alias='main')
+.. code-block:: python
+
+  addr = alice.bind('PUSH', alias='main')
```
Comment (Peque) [inline docs/source/basic_patterns.rst]: Would you mind using a 3-spaces indentation? (instead of 2)

File: docs/source/basic_patterns.rst [hunk 2]
```diff
@@ -49,7 +51,9 @@ There are three things to remark in that line:
 The next interesting line of code is the one in which *Bob* connects to
 *Alice*:
 
-bob.connect(addr, handler=log_message)
+.. code-block:: python
+
+  bob.connect(addr, handler=log_message)
```
Comment (Peque) [inline docs/source/basic_patterns.rst]: Would you mind using a 3-spaces indentation? (instead of 2)

**Comment by RezaBehzadpour** [comment]
Yes, why not.

**Comment by Peque** [comment]
Merged: https://github.com/opensistemas-hub/osbrain/commit/c06bec2c4d5d66801830f530615d540717abee30

Thanks for the contribution! :blush:

Next time, please, configure your email in Git appropriately, to avoid having an ugly email address like `29193529+RezaBehzadpour@users.noreply.github.com`. :stuck_out_tongue_winking_eye:

**Comment by RezaBehzadpour** [comment]
Thanks man. I'll make sure that next time my Git is configured properly. 😇🙏  
Why did you close the PR? Isn't it ready to be merged or you want to merge it in future time? Cause I can't see any changes in the [documentation][1]!

[1]: https://osbrain.readthedocs.io/en/stable/basic_patterns.html

**Comment by Peque** [comment]
The pull request is already merged, see: https://github.com/opensistemas-hub/osbrain/commit/c06bec2c4d5d66801830f530615d540717abee30

I simply squashed the two commits you made and changed the commit subject. You keep the authorship. :blush:

Thanks for reporting that the changes were not displayed in the documentation. It seems Read the Docs was misconfigured. I fixed that and you should now be able to see your changes in the [`latest` version of the documentation](https://osbrain.readthedocs.io/en/latest/basic_patterns.html). It will be included in `stable` once we roll out the next release (no ETA for that).

**Comment by RezaBehzadpour** [comment]
Thanks for your kindness when responding to my comments. 😇🌹🌻
Wish you the best. 👑

**Pull request closed without merging**